include/deal.II/base/vectorization.h

```diff
@@ -426,14 +426,20 @@ class VectorizedArray
    */
   using value_type = Number;
 
-  static_assert(width == 1,
-                "You specified an illegal width that is not supported.");
+  /**
+   * TODO
+   */
+  static constexpr bool is_available = false;
```

Comment:
Shouldn't it be
```suggestion
  static constexpr bool is_available = (width == 1);
```
?

Reply:
Yup. Updated.